pkg/apis/componentconfig/v1alpha1/types.go

```diff
@@ -57,6 +57,9 @@ type DeschedulerConfiguration struct {
 	// IgnorePVCPods sets whether PVC pods should be allowed to be evicted
 	IgnorePVCPods bool `json:"ignorePvcPods,omitempty"`
 
+	// IgnorePodsWithResourceClaims sets whether pods using resource claims should be allowed to be evicted
+	IgnorePodsWithResourceClaims bool `json:"ignorePodsWithResourceClaims,omitempty"`
```

Comment:
Fields from `DeschedulerConfiguration` are no longer used. Only the default evictor options are valid.